### PR TITLE
Plot Target Time in surv Rate with CI [survival]

### PR DESCRIPTION
**Pull request by philipperuiz**
if `pool.replicate == TRUE` with a fixed `target.time` confidence interval are plotted.
The response is in survival rate

### CODE REVIEW

File: R/plot.survData.R [hunk 1]
```diff
@@ -1,16 +1,13 @@
 #' Plotting method for \code{survData} objects
 #'
 #' Plots the number of survivors as a
-#' function of either time and concentration, time only (for a fixed
-#' concentration), concentration only (for a given target time). If both
-#' concentration and target time are fixed, the function additionally plots
-#' the experimental values for the minimum available concentration.
+#' function of either time, time only (for a fixed
```
Comment (pveber) [inline R/plot.survData.R]: this sentence needs to be fixed :o)

File: R/plotDoseResponse.R [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#' Plot Dose-response from raw data
+#' 
+#' Plots the survival rate or reproduction rate as a
```
Comment (pveber) [inline R/plotDoseResponse.R]: remove reproduction rate

File: R/plotDoseResponse.survData.R [hunk 3]
```diff
@@ -0,0 +1,218 @@
+#' @importFrom stats aggregate binom.test
+survConfInt <- function(x, log.scale) {
+  # create confidente interval on observed data for the log logistic
```
Comment (pveber) [inline R/plotDoseResponse.survData.R]: confidence

File: R/plotDoseResponse.survData.R [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+  if (log.scale) ci <- ci[ ,colnames(ci) != 0]
+  
+  return(ci)
+}
+
+
+#' Plotting method for \code{survData} objects
+#'
+#' Plots the survival rate as a
+#' function of either concentration (for a given target time).
```
Comment (pveber) [inline R/plotDoseResponse.survData.R]: remove either

File: R/plotDoseResponse.survData.R [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+#'
+#' @param x an object of class \code{survData}
+#' @param xlab a title for the \eqn{x}-axis (optional)
+#' @param ylab a label for the \eqn{y}-axis
+#' @param main main title for the plot
+#' @param target.time a numeric value corresponding to some observed time in \code{data}
+#' @param style graphical backend, can be \code{'generic'} or \code{'ggplot'}
+#' @param pool.replicate if \code{TRUE}, the datapoints of each replicate are
```
Comment (pveber) [inline R/plotDoseResponse.survData.R]: remove pool.replicate argument

File: R/plotDoseResponse.survData.R [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+#' summed for a same concentration
+#' @param log.scale if \code{TRUE}, displays \eqn{x}-axis in log scale
+#' @param addlegend if \code{TRUE}, adds a default legend to the plot
```
Comment (pveber) [inline R/plotDoseResponse.survData.R]: check if addlegend is still necessary

File: R/plotDoseResponse.survData.R [hunk 7]
```diff
@@ -0,0 +1,218 @@
...
+#' @examples
+#'
+#' library(ggplot2)
+#'
+#' # (1) Load the data
+#' data(zinc)
+#' zinc <- survData(zinc)
+#'
+#' # (2) Plot dose-response
+#' plotDoseResponse(zinc, addlegend = TRUE)
```
Comment (pveber) [inline R/plotDoseResponse.survData.R]: if addlegend is removed, remove this example

File: R/plotDoseResponse.survData.R [hunk 8]
```diff
@@ -0,0 +1,218 @@
...
+#'
+#' @export
+plotDoseResponse.survData <- function(x,
+                                      xlab = "Concentration",
+                                      ylab = "Survival rate",
+                                      main = NULL,
+                                      target.time = NULL,
+                                      style = "generic",
+                                      log.scale = FALSE,
+                                      addlegend = FALSE,
```
Comment (pveber) [inline R/plotDoseResponse.survData.R]: to remove?

File: R/plotDoseResponse.survData.R [hunk 9]
```diff
@@ -0,0 +1,218 @@
...
+                                      remove.someLabels = FALSE,
+                                      pool.replicate = FALSE,
```
Comment (pveber) [inline R/plotDoseResponse.survData.R]: to be removed

File: R/plot.survData.R (hunk 1 above)
Comment (philipperuiz) [inline R/plot.survData.R]: #1fe20b8a

File: R/plotDoseResponse.R (hunk 2 above)
Comment (philipperuiz) [inline R/plotDoseResponse.R]: #526653be

File: R/plotDoseResponse.survData.R (hunk 3 above)
Comment (philipperuiz) [inline R/plotDoseResponse.survData.R]: #15826a2

File: R/plotDoseResponse.survData.R (hunk 4 above)
Comment (philipperuiz) [inline R/plotDoseResponse.survData.R]: #882c3c0d

File: R/plotDoseResponse.survData.R (hunk 5 above)
Comment (philipperuiz) [inline R/plotDoseResponse.survData.R]: #b6ae2c2

File: R/plotDoseResponse.survData.R (hunk 6 above)
Comment (philipperuiz) [inline R/plotDoseResponse.survData.R]: #5e8ff079

File: R/plotDoseResponse.survData.R (hunk 7 above)
Comment (philipperuiz) [inline R/plotDoseResponse.survData.R]: #5e8ff079

File: R/plotDoseResponse.survData.R (hunk 8 above)
Comment (philipperuiz) [inline R/plotDoseResponse.survData.R]: #27d880c

File: R/plotDoseResponse.survData.R (hunk 9 above)
Comment (philipperuiz) [inline R/plotDoseResponse.survData.R]: #27d880c

**Pull request closed without merging**